### PR TITLE
feat(Lightroom): Add `Disable mandatory login` patch

### LINKED ISSUE #725: feat: Bypass Sign in of Adobe Products
### Application

Photoshop Express Photo Editor (https://play.google.com/store/apps/details?id=com.adobe.psmobile)
Lightroom Photo & Video Editor (https://play.google.com/store/apps/details?id=com.adobe.lrmobile)

### Issue

Sign in mandatory to use the app.

### Patch

Patch to disable sign in prompt so that the apps can be used without signing in.

### Motivation

Since these apps cannot be used without sign in, it is bad from a privacy standpoint as Adobe is known to often use privacy invasive features.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

### LINKED ISSUE #928: feat(patch): Adobe Lightroom Patch
### Application

Adobe Lightroom

### Issue

Unlock the premium features in Adobe Lightroom.

### Patch

Unlock premium features like magic eraser, fix and adjust perspective, make enhancements to specific areas of the photo and not the whole picture, etc.

### Motivation

Adobe Photoshop is arguably the most powerful photo editing app on Android and probably the only app besides 'Google photo premium' to have the "magic eraser" tool.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

### PR DESCRIPTION
**Pull request by Aunali321**
closes https://github.com/ReVanced/revanced-patches-template/issues/928
partially closes https://github.com/ReVanced/revanced-patches-template/issues/727

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/lightroom/misc/login/patch/BypassLoginPatch.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import app.revanced.patches.lightroom.misc.login.fingerprint.BypassLoginFingerprint
+
+@Patch
+@Name("Bypass login")
+@Description("Bypasses the requirement to login.")
+@BypassLoginCompatibility
+class BypassLoginPatch : BytecodePatch(listOf(BypassLoginFingerprint)) {
+    override fun execute(context: BytecodeContext): PatchResult {
+        BypassLoginFingerprint.result?.mutableMethod?.apply {
+            val index = implementation!!.instructions.lastIndex - 1
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/lightroom/misc/login/patch/BypassLoginPatch.kt:22]: Please add a comment what this index represents.

File: src/main/kotlin/app/revanced/patches/lightroom/misc/login/fingerprint/BypassLoginFingerprint.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package app.revanced.patches.lightroom.misc.login.fingerprint
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+object BypassLoginFingerprint : MethodFingerprint(
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/lightroom/misc/login/fingerprint/BypassLoginFingerprint.kt:8]: Give this fingerprint a better name

**Comment by oSumAtrIX** [comment]
What do you mean by "partially closes"?

**Comment by Aunali321** [comment]
> What do you mean by "partially closes"?

Adds one of the mentioned request. The rest will be added in my next patch

**Comment by oSumAtrIX** [comment]
Please separate this PR into two feats, can't squash.

**Pull request merged**